web/ui/react-app/src/ExpressionInput.tsx

```diff
@@ -8,8 +8,9 @@ import {
   Input,
 } from 'reactstrap';
 
-import Downshift, { ChildrenFunction, ControllerStateAndHelpers, DownshiftInterface } from 'downshift';
```

Comment:
I just fixed this and other linter errors in master - could you rebase?

Reply:
This is done.